lib/php/Converter.php

```diff
@@ -463,10 +464,17 @@ private function convertTo01(array $data)
                 if (isset($soft['installdate'])) {
                     $soft['install_date'] = $soft['installdate'];
                     unset($soft['installdate']);
-                    $soft['install_date'] = $this->convertDate(
-                        $soft['install_date'],
-                        'Y-m-d'
-                    );
+                }
+
+                if (isset($soft['install_date'])) {
+                    if ($soft['install_date'] == 'N/A') {
+                        unset($soft['install_date']);
+                    } else {
+                        $soft['install_date'] = $this->convertDate(
+                            $soft['install_date'],
+                            'Y-m-d'
+                        );
+                    }
                 }
```

Comment:
Maybe it could be factorized.

1. `Converter::convertDate()` may return `null` if date value is `N/A`.
2. `Converter::convertDate()` may have default `Y-m-d` value for `$format` attribute.
3. If both point 1 and 2 are done, usage could be simplified to:
```suggestion
                if (isset($soft['install_date'])) {
                    if (($convertedDate = $this->convertDate($soft['install_date'])) !== null) {
                        $soft['install_date'] = $convertedDate;
                    } else {
                        unset($soft['install_date']);
                    }
                }
```